src/platform/backends/libvirt/libvirt_virtual_machine.cpp

```diff
@@ -332,6 +342,16 @@ int mp::LibVirtVirtualMachine::ssh_port()
     return 22;
 }
 
+void mp::LibVirtVirtualMachine::ensure_vm_is_running()
```

Comment:
Because in the qemu backend, we are checking that the qemu's `QProcess` is actually running and didn't die for some reason.  Here, we are relying on `libvirtd` to do stuff and have no control of the qemu process itself.  I suppose it wouldn't hurt to add `ensure_vm_is_running()` in `LibVirtVirtualMachine::ssh_hostname()`, so I'll do that.